syllabus/syllabus-app/src/webapp/syllabus/edit_bulk.jsp

```diff
@@ -208,15 +208,15 @@
 						</h:panelGroup>
 					</h:panelGroup>
 					<h:panelGroup>
-						<h:selectOneRadio id="radioSingleItem" value="#{SyllabusTool.bulkEntry.addSingleItem}" styleClass="radioSingleItem radioOption radioOptionSelected">
+						<h:selectOneRadio id="radioSingleItem" value="#{SyllabusTool.bulkEntry.addSingleItem}" styleClass="radioSingleItem radioOption radioOptionSelected" style="width:100%">
```

Comment:
Update this and others as well

Reply:
Thanks! I updated the code and pushed it.